### PR TITLE
Html basics

### PR DESCRIPTION
**Pull request by HappyJuliana**
image

в следующий раз обещаю чаще коммитить :)
а то в этот раз вышло все до кучи, но было уже поздно что-то менять

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    </p>
+    <p>In HTML, each element is made up of 3 parts:
+    </p>
+    <ol>
+        <li>The opening tag</li>
+        <li>The closing tag</li>
+        <li>The actual element</li>
+      </ol>
+
+    <h3>Why should you learn HTML?
```
Comment (IvanCeban) [inline index.html]: Get used to keep the code always aligned. 
So the opening and closing tags should be on the same column. 
If you have element in 1 line, then opening and closing tags should be in 1 line.

Comment (IvanCeban) [reply index.html]: Remove empty lines.

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Some cleanup is required, overall Great job!

File: index.html (hunk 1 above)
Comment (IvanCeban) [reply index.html]: The main point of my comments is to keep the code as clean as possible

Comment (HappyJuliana) [reply index.html]: Thanks! I've made some corrections so now the code looks "clean" (as I think)

**Review by IvanCeban — APPROVED** [review]

**Pull request merged**